hexrd/ui/hexrd_config.py

```diff
@@ -363,6 +363,8 @@ def _save_state_to_settings(self, state, settings):
     def _load_state_from_settings(self, settings):
         state = {}
         for name, default in self._attributes_to_persist():
+            if name == 'azimuthal_overlays':
+                continue
```

Comment:
Can you leave a comment here that we are doing this so that `azimuthal_overlays` get saved to state but do not get saved to the QSettings in between sessions?

Reply:
Done!